### ISSUE TITLE
Incentive for contribution and participation

### ISSUE THREAD

**Issue by mrclo90** [Blockchain-at-Yonsei/voting-proposal#1]
**Incentive for contribution and participation**
* Proposal: Kihun Yun
* Team: Education Circle

**Proposal Description**
다음 표와 같은 항목에 대해 주어진 rewards (B@Y)를 부여한다. 

<img width="756" alt="스크린샷 2021-10-07 오후 5 13 20" src="https://user-images.githubusercontent.com/63194399/136346053-3f88a415-0b13-4167-b890-c27a7c576c42.png">

* Voting: 투표에 참여하면 투표 결과에 상관 없이 2 B@Y의 인센티브를 받는다.
* Proposal: Proposal을 제안하여 투표 결과 통과된 경우에 한해 10 B@Y의 rewards를 지급한다.
* Question rewards: 세션 중 가장 좋은 질문을 한 사람 1명에게 주어지며 세션 진행자가 선정한다.
* Bug Bounty: DAO 운영에 필요한 스마트 컨트랙트에서 오류를 발견한 사람에게 10 B@Y를 지급한다. 
* Grant Proposal: B@Y DAO에 기여하기 위해서 grant가 필요한 사람은 예산과 활동 계획을 제시하고 투표를 통해 요청한 grant를 할당받게 된다. grant는 한 달을 주기로 할당되고 grant를 요청한 팀은 한 달을 주기로 진행 사항을 board member에게 공지할 의무를 진다. 
* Session: DAO 멤버들을 대상으로 세션을 진행한 사람은 1시간 단위로 30 B@Y를 지급한다. 
* Workshop: 워크숍을 주최한 사람들은 1시간 단위로 50 B@Y를 지급받는다. 
* Resource contribution은 글, ppt등 learning materials를 제작한 사람에게 주어진다. 단, session, workshop, 그리고 grant를 신청한 경우 해당 rewards를 지급하지 않는다. 또한  퀄리티가 지나치게 저조할 시 Education circle 판단에 따라 지급하지 않을 수 있다.

**Specification**
<img width="597" alt="스크린샷 2021-10-07 오후 5 15 29" src="https://user-images.githubusercontent.com/63194399/136346297-bda177e2-1e51-4792-8438-65f191c6badf.png">

* 현재 smart contract로 구현되어 있지 않기 때문에 일주일 간격으로 education circle 및 기타 주체가 지급 여부를 검토한 뒤 이를 treasury에 인계한다. Treasury는 정해진 B@Y를 대상자에게 지급한다. 
* 추후 smart contract가 구현된다면 voting과 proposal은 인간의 개입없이 자동으로 지급되도록 한다. 

**Benefits**
  해당 proposal은 2가지 측면의 기대효과를 가진다. 
   * B@Y 멤버의 활발하고 적극적인 참여 요인을 제공한다.
   * B@Y DAO에 contribution을 기여한 사람에게 적절한 보상을 제공할 수 있다. 

**Drawbacks**
  * 아직 smart contract가 제대로 구현되지 않았기 때문에 incentive 부여를 위해서 Education Circle의 중간 개입이 필요하다. 
  * 지나친 토큰 보유 개수의 격차를 불러올 가능성이 존재한다.

**Comment by jihwan97-hash** [comment]
@mrclo90  Grant proposal 항목에서 grant를 할당받는다는게 B@Y token을 지급받는다는 뜻이죠?

**Comment by mrclo90** [comment]
> @mrclo90 Grant proposal 항목에서 grant를 할당받는다는게 B@Y token을 지급받는다는 뜻이죠?

네 맞습니다.

**Comment by Star-Apostrophe** [comment]
Proposal 보다 Bug bounty가 좀 더 많은 토큰을 얻어야 한다고 생각합니다. 아무래도 전문성과 시간을 Proposal보다 더 많이 요구하니 좀 더 많은 인센티브가 있어야 할 것 같습니다.

**Comment by mrclo90** [comment]
넵 타당한 의견인 것 같습니다. 혹시 그럼 bug bounty의 인센티브로 얼만큼 지급하는 것이 좋다고 생각하시나요?

**Comment by yoontaeung** [comment]
추가적인 건의사항이 있는데요, 준비한 material이나 세션 등의 퀄리티에 따라서 보너스 토큰이 주어지면 어떨까요? 예를 들어 education cycle의 투표에 따라서 정해진 기본 토큰 + 기본토큰의 10%? 를 팁으로 주면 더 큰 동기부여가 될 것 같습니다. ㅎㅎ

**Comment by mrclo90** [comment]
네 좋은 의견 감사합니다! 확실히 보너스 토큰 제도가 있다면 퀄리티를 향상시킬 중요한 유인책이 될 것 같아요. 퀄리티 평가방법이 조금 고민인데 말씀하신 방법도 괜찮은 방안인 것 같고 이 부분 조금 더 생각해서 바로 공유드리겠습니다. 혹시 관련해서 추가적인 아이디어 있으시면 또 말씀해주세요:)

**Comment by jihwan97-hash** [comment]
> 추가적인 건의사항이 있는데요, 준비한 material이나 세션 등의 퀄리티에 따라서 보너스 토큰이 주어지면 어떨까요? 예를 들어 education cycle의 투표에 따라서 정해진 기본 토큰 + 기본토큰의 10%? 를 팁으로 주면 더 큰 동기부여가 될 것 같습니다. ㅎㅎ

@yoontaenng 태웅님이 혹시 중간고사 이후에 퀄리티를 판단할 수 있는 rubric을 만들어서 투표할 수 있는 방안을 proposal로 제안해보면 어떠실까요? voting proposal 제안 시에 토큰도 얻을 수 있고 새롭게 의견도 제안할 수 있어서 학회에 도움이 될 것 같습니다!
지금 진행중인 포럼 논의도 좋고, 많은 분들이 다양한 방면으로 proposal을 제안해주셨으면 좋겠습니다 :)

**Comment by mrclo90** [comment]
네 직접 proposal을 내면 위에 제시된 incentive도 얻을 수 있고 좋을 것 같습니다. 태웅님이 제안해주신 내부 투표 방안도 괜찮은 방법인 것 같은데 edu circle만이 판단을 내리기엔 다소 자의적일 수 있으니 범위를 확장해서 board의 투표로 진행하는 건 어떤가요?

**Comment by yoontaeung** [comment]
> > 추가적인 건의사항이 있는데요, 준비한 material이나 세션 등의 퀄리티에 따라서 보너스 토큰이 주어지면 어떨까요? 예를 들어 education cycle의 투표에 따라서 정해진 기본 토큰 + 기본토큰의 10%? 를 팁으로 주면 더 큰 동기부여가 될 것 같습니다. ㅎㅎ
> 
> @yoontaenng 태웅님이 혹시 중간고사 이후에 퀄리티를 판단할 수 있는 rubric을 만들어서 투표할 수 있는 방안을 proposal로 제안해보면 어떠실까요? voting proposal 제안 시에 토큰도 얻을 수 있고 새롭게 의견도 제안할 수 있어서 학회에 도움이 될 것 같습니다! 지금 진행중인 포럼 논의도 좋고, 많은 분들이 다양한 방면으로 proposal을 제안해주셨으면 좋겠습니다 :)

네 제가 좀 더 생각해보고 proposal 올려보겠습니다!

**Comment by Star-Apostrophe** [comment]
> 넵 타당한 의견인 것 같습니다. 혹시 그럼 bug bounty의 인센티브로 얼만큼 지급하는 것이 좋다고 생각하시나요?

이 부분은 voting proposal에 올려 다 같이 합의하는 것이 좋을 것 같습니다.

**Comment by ihave-been** [comment]
확인했습니다 :)
전반적으로 공정하게 잘 짜여진 것 같습니다

저는 지금 논의 되고 있는 토큰 분배와 관련해서 한가지 저희가 이야기할 부분이 있다고 생각합니다
저희가 여러 선택의 상황에서 결국 하나의 방향으로 선택하게 될 것인데
여기서 여러 의견을 듣는 것은 좋습니다
하지만 구성원 간의 먼저 어떤 목표인지 확인하고 정하는 것은 그 전에 필요한 것 같습니다

Star 님과 yoontaenng님 의견에 있어서도 저희가 각자 어떻게 하자고 판단을 할 것이지만 
이 전에 앞서 우리가 어떤 것을 추구해야할 지 논의하는 것이 중요할 것 같습니다

예를 들어 우리는 지금은 교육을 위한 학회로 운영을 논의하고
이 교육의 방향은 후에 DAO를 기술적으로나 이념적으로 바른 방향으로 잘 굴러가게 하는 것으로 합의를 한다면 
세션과 워크샵에 이정도의 토큰을 분배해야한다

의 순서가 맞다고 생각합니다

조금 말이 길어졌는데 
토큰 분배등 상황에 대한 판단을 할 때 사람마다 기준이 다를 수 있습니다 
이럴 때 의견을 모으기 위한 하나의 큰 줄기 
우리가 모인 목적이 이것이니 이 방향에 맞게 생각한다면 이렇게 가는 것이 맞다 라고 이야기할 논의 된 목표가 아직 없는 것 같습니다
보상을 나누는 것은 좋지만 그 전에 우리가 어떤 부분을 집중하고 어떤 가치판단의 근거하에 운영해갈지 논의해야하지 않을까?
라고 생각합니다

이 부분에 있어서 개인적으로는
중앙화된 방식이 초창기에는 효율적이라고 생각하는데 이 내용에 대해서도 기회가 된다면 이야기해보면 좋을것 같습니다

감사합니다

**Comment by Star-Apostrophe** [comment]
> 확인했습니다 :) 전반적으로 공정하게 잘 짜여진 것 같습니다
> 
> 저는 지금 논의 되고 있는 토큰 분배와 관련해서 한가지 저희가 이야기할 부분이 있다고 생각합니다 저희가 여러 선택의 상황에서 결국 하나의 방향으로 선택하게 될 것인데 여기서 여러 의견을 듣는 것은 좋습니다 하지만 구성원 간의 먼저 어떤 목표인지 확인하고 정하는 것은 그 전에 필요한 것 같습니다
> 
> Star 님과 yoontaenng님 의견에 있어서도 저희가 각자 어떻게 하자고 판단을 할 것이지만 이 전에 앞서 우리가 어떤 것을 추구해야할 지 논의하는 것이 중요할 것 같습니다
> 
> 예를 들어 우리는 지금은 교육을 위한 학회로 운영을 논의하고 이 교육의 방향은 후에 DAO를 기술적으로나 이념적으로 바른 방향으로 잘 굴러가게 하는 것으로 합의를 한다면 세션과 워크샵에 이정도의 토큰을 분배해야한다
> 
> 의 순서가 맞다고 생각합니다
> 
> 조금 말이 길어졌는데 토큰 분배등 상황에 대한 판단을 할 때 사람마다 기준이 다를 수 있습니다 이럴 때 의견을 모으기 위한 하나의 큰 줄기 우리가 모인 목적이 이것이니 이 방향에 맞게 생각한다면 이렇게 가는 것이 맞다 라고 이야기할 논의 된 목표가 아직 없는 것 같습니다 보상을 나누는 것은 좋지만 그 전에 우리가 어떤 부분을 집중하고 어떤 가치판단의 근거하에 운영해갈지 논의해야하지 않을까? 라고 생각합니다
> 
> 이 부분에 있어서 개인적으로는 중앙화된 방식이 초창기에는 효율적이라고 생각하는데 이 내용에 대해서도 기회가 된다면 이야기해보면 좋을것 같습니다
> 
> 감사합니다

동의합니다. 확실히 목적을 먼저 정해야 구체적으로 각 항목 당 토큰을 얼마나 분배해야 할 지 정할 수 있을 것 같습니다. 현재는 구체적인 수치로 얼마를 부여한다 보다는 각 항목의 대, 소 정도만 구분한 후 목적에 따라 구체적인 수치를 정하는게 좋을 것 같습니다.

**Comment by mrclo90** [comment]
@ihave-been 네 좋은 의견 감사합니다! 
말씀해주신대로 B@Y의 목적과 방향에 대해 구성원의 합의를 거치고 이를 기반으로 원칙과 기준을 확립하는 것이 바람직하다고 생각합니다.
 
일단 제가 forum에 글을 먼저 올렸기 때문에, 제가 생각했었던 B@Y의 목적을 밝혀야 할 것 같습니다. 저는 B@Y의 목적이 Learn to Earn이라고 생각했습니다. B@Y는 블록체인을 공부하고 알아가고 싶은 학생들이 모인 커뮤니티입니다. 그러나 그 커뮤니티가 특정 누군가의 희생에 의해서만 유지되고 전적으로 이에 의지해 움직이는 것은 안된다고 생각합니다. 그런 커뮤니티는 지속성이 없고 참여자와 기여자 모두 진정한 성장과 만족을 누릴 수 없기 때문입니다. 따라서 적절한 인센티브를 통해 구성원 스스로 커뮤니티에 기여하는 시스템을 구축하고자 했습니다. 이를 통해 커뮤니티의 저변을 넓히고 궁극적으로 **'블록체인에 정말 관심있고 흥미 있는 사람들이 자발적으로 배우고 커뮤니티에 기여하고 논의하며 세상을 바꾸는 새로운 아이디어를 만들어 나가는 곳'** 을 만들고 그들과 계속, 같이, 함께하고 싶기 때문입니다. 

이러한 목적과 배경으로 기여에 대한 인센티브 제도에 대해 forum에 글을 게재하게 되었습니다. 인센티브 제도를 설계하면서 고안했던 3가지 원칙은 다음과 같습니다. 

> 기여에 대한 적절한 보상 

     기여자는 B@Y에 자신의 리소스를 투입하는 만큼, 이에 상응하는 보상이 필요하다.

> 자발적 참여 촉진 

    인센티브는 추후 추가적인 기여의 등장을 촉진할 만큼 충분히 매력적이어야 한다.

> 탈중앙화 커뮤니티 시스템 안정성 

    인센티브 매커니즘은 B@Y의 거버넌스, 토큰 이코노미 등의 기본적 시스템을 해치지 않는 수준이어야 한다. 

이 3가지 원칙과 모두에게 처음 분배된 토큰의 개수가 200이라는 점을 고려하여 위 제시된 수치가 등장하게 되었습니다.

위 게시글에 제가 생각했던 이러한 목적과 원칙을 같이 공유드렸어야 하는데 미처 신경 쓰지 못했습니다. 그리고 이를 다루어 주신 것에 대해 정말 감사드립니다. 위 게시글에서 다룬 내용은 B@Y의 목적과 방향에 대한 커뮤니티의 의견을 좀 더 반영하고 논의를 거친 뒤 proposal로 제안하도록 하겠습니다. 

누군가 독단으로 모든 것을 결정하는 것은 궁극적으로 이전에 실패했던 "누군가의 희생으로 움직이고 다수가 수동적으로 뒤따르는" 모델에서 벗어나기 어려울 것이라 생각됩니다. B@Y는 탈중앙화 자치 조직입니다. 이를 위해서는 최대한 커뮤니티 모두의 자유롭고 적극적인 의견 개진과 합의를 통해 최선의 방법을 도출하는 것이 꼭 필요합니다. 따라서 앞으로도 지금처럼 포럼을 통해, 디스코드를 통해, 그리고 가능하시다면 수요일 타운홀 미팅을 통해 커뮤니티 모두의 목소리를 더 많이 듣고 싶습니다. 그리고 이를 통해 말씀해주신 "우리가 모인 목적이 이것이니 이 방향에 맞게 생각한다면 이렇게 가는 것이 맞다"에 대한 커뮤니티 구성원들의 의지와 의욕의 일치가 이루어지기를 희망합니다. 

감사합니다!

**Comment by ihave-been** [comment]
@mrclo90 

답변 감사합니다 :) 
넵! 타운홀 미팅에도 참여해 더 의견내어 좋은 커뮤니티 되도록 함께 노력하겠습니다!
